zerver/openapi/zulip.yaml

```diff
@@ -18979,6 +18999,39 @@ paths:
                           Present if `realm_user` is present in `fetch_event_types`.
 
                           Whether the current user is a [guest user](/api/roles-and-permissions).
+                      show_billing:
+                        type: boolean
+                        description: |
+                          Present if `realm_user` is present in `fetch_event_types`.
```

Comment:
The code has these present for the `realm` event type, which seems more correct than `realm_user`, so I think yuo've got a copy/paste typo in each of these lines.